projects/xiph.org/flac/package.yml

```diff
@@ -0,0 +1,53 @@
+distributable:
+  url: https://downloads.xiph.org/releases/flac/flac-{{version}}.tar.xz
+  strip-components: 1
+
+versions:
+  github: xiph/flac/tags
```

Comment:
use `xiph/flag/releases/tags` or we don't get updates

Reply:
applied.